stack.yaml

```diff
@@ -1,6 +1,3 @@
-flags:
-  path:
-    validity: true
```

Comment:
Can you add back the flag and instead use the `:test` target in the appveyor.yml? This way validity-test should still run on Travis.